packages/pirania-app/files/www/cgi-bin/pirania/handle_voucher

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/lua
+
+local http = require('luci.http')
+local logic = require('voucher.logic')
+local redirect_page = require('voucher.utils').redirect_page
+local uci_cursor = require('uci').cursor()
+local dba = require('voucher.db')
+local query_string = os.getenv("QUERY_STRING")
+-- send("Status: 302 \r\n")
+print("Content-type: text/html \n\n")
+params = http.urldecode_params(query_string)
+local res = logic.getIpv4AndMac()
+local valid = logic.check_mac_validity(res.mac)
+local noJs = params['nojs']
+
+local url_authenticated = uci_cursor:get("pirania", "base_config", "url_authenticated")
+local url_fail = uci_cursor:get("pirania", "base_config", "url_fail")
+local url_info = uci_cursor:get("pirania", "base_config", "url_info")
+
+local url
+
+if (valid < 1) then
+  local db = dba.load(config.db)
```

Comment:
Here the `config` variable is not loaded from anywhere. Is this working? If it is working then it may be being populated as a global variable by mistake somewhere (but I can't find the place)

Reply:
Good catch @spiccinini! None of the `cgi-bin` handles are importing `config`.

Can we merge #672 before fixing this?

Reply:
Yes, I aproved the PR (needs one more aprove though)

Reply:
:+1: